src/types/homepage.ts

```diff
@@ -92,6 +92,18 @@ export interface TextcardsSection {
   cards: TextCardItem[]
 }
 
+export interface Infobox {
```

Comment:
nit: abit weird to not prefix with `InfoCol` when it is done so elsewhere

Reply:
refactored